var/spack/repos/builtin/packages/py-traceback2/package.py

```diff
@@ -16,6 +16,7 @@ class PyTraceback2(PythonPackage):
 
     depends_on('py-setuptools', type='build')
     depends_on('py-pbr', type='build')
+    depends_on('py-linecache2', type='build')
```

Comment:
```suggestion
    depends_on('py-linecache2', type='run')
```
If it's a build dependency, how did `py-traceback2` successfully build in the first place? Seems more like a run dependency to me?